test/integration-test.mk

```diff
@@ -58,14 +60,31 @@ test-addon-integration: ensure-kubebuilder-tools
 # - unmanaged_appliedwork_test.go, this test mainly focus on switching the hub kube-apiserver
 # - manifestworkreplicaset_test.go, this test needs to update the work status with the hub work client,
 #   cloud events work client does not support it. (TODO) may add e2e to for mwrs.
-test-cloudevents-integration: ensure-kubebuilder-tools build-work-integration
+test-cloudevents-work-mqtt-integration: ensure-kubebuilder-tools build-work-integration
 	./work-integration.test -ginkgo.slow-spec-threshold=15s -ginkgo.v -ginkgo.fail-fast \
 		-ginkgo.skip-file manifestworkreplicaset_test.go \
 		-ginkgo.skip-file executor_test.go \
 		-ginkgo.skip-file unmanaged_appliedwork_test.go \
 		-test.driver=mqtt \
 		-v=4 ${ARGS}
-.PHONY: test-cloudevents-integration
+.PHONY: test-cloudevents-work-mqtt-integration
+
+test-cloudevents-work-grpc-integration: ensure-kubebuilder-tools build-work-integration
+	./work-integration.test -ginkgo.slow-spec-threshold=15s -ginkgo.v -ginkgo.fail-fast \
+		-ginkgo.skip-file manifestworkreplicaset_test.go \
+		-ginkgo.skip-file executor_test.go \
+		-ginkgo.skip-file unmanaged_appliedwork_test.go \
+		-test.driver=grpc \
+		-v=4 ${ARGS}
+.PHONY: test-cloudevents-work-grpc-integration
+
+test-cloudevents-registration-grpc-integration: ensure-kubebuilder-tools build-registration-integration
+	./registration-integration.test -ginkgo.slow-spec-threshold=15s -ginkgo.v -ginkgo.fail-fast \
```

Comment:
more description on this.

Reply:
using a single grpc test to instead of this